blockly-core/tests/phantomwrapper.js

```diff
@@ -0,0 +1,76 @@
+var fs = require('fs');
+var WebPage = require('webpage');
+
+var DEFAULT_STYLE = "\033[39m";
+var GREEN = "\033[32m";
+var RED = "\033[31m";
```

Comment:
Tried that actually.  It did color my output, but also generated warnings that I couldn't get rid of - something about us running the phantomjs executable directly, not using node the normal way. I figure plan B here is good enough for now.